cognite/neat/_session/_physical.py

```diff
@@ -40,6 +48,14 @@ def yaml(self, io: Any, exclude_none: bool = False) -> None:
         """Write physical data model to YAML file"""
 
         file_path = NeatReader.create(io).materialize_path()
-        writer = DMSTableExporter(exclude_none=exclude_none)
+        writer = DMSYamlExporter(exclude_none=exclude_none)
+
+        return self._store.write_physical(writer, file_path=file_path)
+
+    def excel(self, io: Any, exclude_none: bool = False) -> None:
+        """Write physical data model to Excel file"""
+
+        file_path = NeatReader.create(io).materialize_path()
+        writer = DMSExcelExporter(exclude_none=exclude_none)
```

Comment:
Given the comments above, then the Excel importer will always include all columns no matter what. Thus it is not exposed as an option. 
```suggestion
    def excel(self, io: Any) -> None:
        """Write physical data model to Excel file"""

        file_path = NeatReader.create(io).materialize_path()
        writer = DMSExcelExporter()
```

Reply:
Updated